AdminWebsite/AdminWebsite.IntegrationTests/Controllers/ControllerTestsBase.cs

```diff
@@ -20,19 +23,27 @@ public class ControllerTestsBase
     {
         private TestServer _server;
         private string _bearerToken = String.Empty;
-        
+
+        protected Mock<IUserAccountService> userAccountService { get; private set; }
+
         [OneTimeSetUp]
         public void OneTimeSetup()
         {
             var webHostBuilder =
                 WebHost.CreateDefaultBuilder()
                     .UseEnvironment("Development")
                     .UseKestrel(c => c.AddServerHeader = false)
-                    .UseStartup<Startup>();
+                    .UseStartup<Startup>().ConfigureServices(AddMocks);
             _server = new TestServer(webHostBuilder);
             GetClientAccessTokenForBookHearingApi();
         }
 
+        private void AddMocks(IServiceCollection obj)
```

Comment:
As mentioned, if this doesn't work anyway, let's revert it and do the test as unit test only.

Reply:
moved to unit test